packages/combo-box/test/external-filtering.test.js

```diff
@@ -0,0 +1,184 @@
+import { expect } from '@esm-bundle/chai';
+import { fire, fixtureSync } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import './not-animated-styles.js';
+import '../vaadin-combo-box.js';
+import { getAllItems } from './helpers.js';
+
+describe('external filtering', () => {
+  let comboBox, overlay;
+
+  function setInputValue(value) {
```

Comment:
Let's move this function to `helpers.js` as it's not used in several test files.

Reply:
Done.